beets/library.py

```diff
@@ -715,6 +715,71 @@ def move(self, copy=False, link=False, basedir=None, with_album=True):
 
     # Templating.
 
+    def _legalize_partial(self, path, fragment, replacements):
```

Comment:
That sounds good. I did initially think about putting them in utils, alongside sanitize and truncate.